packages/jest-dom-mocks/src/match-media.ts

```diff
@@ -3,35 +3,32 @@ export interface MediaMatching {
 }
 
 export default class MatchMedia {
-  originalMatchMedia: (mediaQuery: string) => MediaQueryList;
-  private isUsingMockMatchMedia = false;
+  originalMatchMedia: ((mediaQuery: string) => MediaQueryList) | null = null;
```

Comment:
By correctly typing `originalMatchMedia` as either a function or null (and setting its initial value to null) we can remove the concept of the `isUsingMockMatchMedia` property.

This is because we can consider that a mock is being used if `originalMatchMedia` is not null